client/sites/components/site-preview-pane/dotcom-preview-pane.tsx

```diff
@@ -109,7 +110,7 @@ const DotcomPreviewPane = ( {
 			},
 			{
 				label: __( 'Logs' ),
-				enabled: isActiveAtomicSite,
+				enabled: isActiveAtomicSite && ! config.isEnabled( 'performance-profiler/logged-in' ),
```

Comment:
@fushar what's the context for this flag? Also, should we check for the `untangling/hosting-menu` flag here?

Reply:
🤦 wrong copy-paste, fixed, thanks!